iFixit/checkout-ui-extensions
# Add Estimated Delivery Date to Checkout Attributes

## Overview

Basically just replicate the functionality from `shopify-v2's`[ checkout-attributes.ts](https://github.com/iFixit/shopify-v2/blob/80abea2a1865cbab9ae79d4929e26d7d7bd4a9e6/src/scripts/checkout/checkout-attributes.ts#L120)

Im not 100% sure this code is fullproof but we seem to have been using for a long time so its probably fine
Should set the estimate to 1 week if there is no estimated data.

## QA

Make sure that the estimated date shows up in the order's attributes, (similar to #54) and its accurate. If the shipping doesn't have an estimate attached to it, the estimate should be 1 week. To check this, you will probably need to set your delivery address to somewhere in europe, ive been using germany.

Closes: #40
